_methods/compensation.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: card
+title: Compensation
+permalink: /fundamentals/compensation/
+redirect_from:
+  - /compensation/
+description: Offering usability test or user research participants gifts to encourage participation and to thank them for their time.
+category: Fundamentals
+what: Offering usability test or user research participants gifts to encourage participation and to thank them for their time.
```

Comment:
```suggestion
what: Offering usability test or user research participants compensation to encourage participation and to thank them for their time.
```

Reply:
@MelissaBraxton thoughts on this change or making it gifts and compensation for clarity?

Reply:
I'm good with removing "gifts" throughout to improve clarity, rather than renaming the section. What do you think?

Reply:
Sounds good to me!